src/NuGet.Services.ServiceBus/NuGet.Services.ServiceBus.csproj

```
@@ -12,11 +12,14 @@
     <PackageReference Include="Microsoft.Extensions.Logging">
       <Version>2.2.0</Version>
     </PackageReference>
+	<PackageReference Include="Microsoft.Identity.Client">
+		<Version>4.25.0</Version>
```

The spacing is off here:

```suggestion
      <Version>4.25.0</Version>
```